### PR TITLE
signatures docs

### PR DESCRIPTION
**Pull request by bjornj12**
Initial Draft for Webhook signature docs. 

Creating these docs before starting to implement, to make sure our customers agree with the approach we take

### CODE REVIEW

**Comment by hugihlynsson** [comment]
It looks like mdx thinks some text is a component, we might have to escape those characters somehow

File: pages/workspace/tracking-plan/webhook-signing.mdx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+### Verify signature header
+
+1. Extract the timestamp and signature from the header
+
+Split the header on comma(,) as seperator to get a list of elements, Then split each element using the = to get a key and value dict. 
+You should now have a dict of 2 key value pairs, 
+```{
+    ts:1651677541,
+    v1:30703093db3c532c6f0ab1f1f9eb80b134f5bf6e2ff7064c403aab7582df0667
+}```
```
Comment (bragur) [inline pages/workspace/tracking-plan/webhook-signing.mdx:49]: Should this be like this? Don’t know if this matters in the grand scheme of things but it seems to mess up the github preview of it. Sorry, don’t know how to escape the code block out
```suggestion
\```
{
    ts:1651677541,
    v1:30703093db3c532c6f0ab1f1f9eb80b134f5bf6e2ff7064c403aab7582df0667
}
\```
```

File: pages/workspace/tracking-plan/webhook-signing.mdx [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+The `signed_payload` string is created by concatinating
+ - The timestamp (string)
+ - The character `.`
+ - The JSON payload (request.body)
+
+ Store this value in a variable for the next step.
+
+ 3. Determine the expected signature
+
+Compute an HMAC using the SHA256 has function. Use the webhooks endpoint signing secret from <your publishing page> as the key, and use the `signed_payload` string as the message.
```
Comment (bragur) [inline pages/workspace/tracking-plan/webhook-signing.mdx:62]: Hmm is that also meant to be a link?
```suggestion
Compute an HMAC using the SHA256 hash function. Use the webhooks endpoint signing secret from _your publishing page_ as the key, and use the `signed_payload` string as the message.
```

File: pages/workspace/tracking-plan/webhook-signing.mdx [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+Avo-Signature:
+ts=1651677541,
+v1=30703093db3c532c6f0ab1f1f9eb80b134f5bf6e2ff7064c403aab7582df0667
+```
+
+Avo generates signatures using a hash-based message authentication code (HMAC) with SHA-256.
+
+### Verify signature header
+
+1. Extract the timestamp and signature from the header
```
Comment (bragur) [inline pages/workspace/tracking-plan/webhook-signing.mdx:42]: Do we have h4? (####)

**Review by bragur — CHANGES_REQUESTED** [review]
Nice! Couple of fixed needed though

**Comment by hugihlynsson** [comment]
#### 📝 Changed routes:


| [**…g-plan/webhook-signing**](https://docs-ipakqxcrk.avo-dev.app/workspace/tracking-plan/webhook-signing) |
|:-:|
|<a href="https://docs-ipakqxcrk.avo-dev.app/workspace/tracking-plan/webhook-signing">Screenshot of /workspace/tracking-plan/webhook-signing</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/OOHr2gTNBGehVy8KWWAWbXaDET_BA49bNp_vSALBDPObzKjHZ9B_QuDPbQ8s3UoL1dww79HsbS0-kJlNptNVCQ.png">(view full size)</a> |







Commit 4dd98aee0d7aa275bdaf96e6f18f9d0ab0201bd6 (https://docs-ipakqxcrk.avo-dev.app).

**Review by hugihlynsson — APPROVED** [review]

**Review by bragur — APPROVED** [review]

**Pull request merged**